src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -93,8 +92,10 @@ public CaseSearchResult searchCases(
             + "search results, please state the alias fields to be returned in the _source property for e.g."
             + " \"_source\":[\"alias.customer\",\"alias.postcode\"]",
             required = true)
-        @RequestBody String jsonSearchRequest) {
+        @RequestBody String jsonSearchRequest,
+        @RequestParam(value = "data_classification", required = false) Boolean dataClassification) {
 
+        dataClassification = (dataClassification == null) ? true : dataClassification.booleanValue();
```

Comment:
```suggestion
        @RequestParam(value = "data_classification", defaultValue = "true") boolean dataClassification) {
```
If this works then we can just let Spring handle it and then pass the primitive everywhere